.rubocop.yml

```
@@ -130,5 +127,4 @@ Layout/EndAlignment:
 
 # Use my_method(my_arg) not my_method( my_arg ) or my_method my_arg.
 Lint/RequireParentheses:
-  Enabled: true
-
+  Enabled: true
```

```suggestion
  Enabled: true

```

Missing EOF new line

---

Thanks, I have since updated my vscode settings so it will now do it automatically!